Makefile

```
@@ -0,0 +1,13 @@
+
+build:
+	docker compose build
+	docker compose run --rm go go mod tidy
+run:
+	docker compose up -d
+run-go:
+	docker compose up go db -d
```

```suggestion
	docker compose up db -d
	docker compose run --rm go go run main.go
```
`run-go`はおそらく、db立ち上げた上で、Goコンテナを立ち上げたいと思われるので、こうなるかと思います。